webapplication/user/admin.py

```diff
@@ -26,14 +26,16 @@ def save(self, commit=True):
 
 class UserAdmin(BaseUserAdmin):
     form = UserForm
-    list_display = ('username', 'email', 'is_staff', 'is_active', 'is_superuser', 'area_limit_ha')
+    list_display = ('username', 'email', 'is_staff', 'is_active', 'is_superuser', 'area_limit_ha', 'receive_news')
+    list_filter = ('receive_news', )
```

Comment:
This overrides the default filter https://github.com/django/django/blob/main/django/contrib/auth/admin.py#L77
Add at least ("is_staff", "is_active", "groups") too

Reply:
fixed